frontend/src/components/Form.js

```diff
@@ -61,29 +95,37 @@ export default function Form() {
     * @param {Event} event
    */
   function handleFormSubmission(event) {
-    // TODO(#34): Validate input values (before allowing to submit)
+    const currentTimeInMilliseconds = new Date().getTime();
+    if (validateFormInputs(
+        role,
+        productArea,
+        duration,
+        timeAvailableUntil.getTime(),
+        currentTimeInMilliseconds)) {
```

Comment:
It looks like all the following code in the function is for this if? If so, please flip the condition and return early if false. This can prevent my question of "where the if ends".

Reply:
Thanks for the suggestions, done!